HTTPServerProject.Tests/HTTPServerProxyTests/TestClient.cs

```diff
@@ -0,0 +1,16 @@
+using HTTPServerProxy.Client;
+
+namespace HTTPServerProxyTests.Client;
+
+public class TestProxyClient : IProxyClient
+{
+    List<string> stream = new List<string>();
+    public TestProxyClient()
```

Comment:
I don't believe the empty constructor is necessary.

Reply:
Agreed.